rules/windows/process_creation/win_xsl_script_processing.yml

```diff
@@ -15,10 +15,21 @@ detection:
     - Image|endswith: '\wmic.exe'
       CommandLine|contains: '/format'     # wmic process list /FORMAT /?
     - Image|endswith: '\msxsl.exe'
-  condition: selection
+  false_positives:
+    - CommandLine|contains: '/Format:List'
+    - CommandLine|contains: '/Format:htable'
+    - CommandLine|contains: '/Format:hform'
+    - CommandLine|contains: '/Format:table'
+    - CommandLine|contains: '/Format:mof'
+    - CommandLine|contains: '/Format:value'
+    - CommandLine|contains: '/Format:rawxml'
+    - CommandLine|contains: '/Format:xml'
+    - CommandLine|contains: '/Format:csv'
+  condition: selection and not false_positives
```

Comment:
Simply
```yaml
  false_positives:
    CommandLine|contains: 
      - '/Format:List'
      - '/Format:htable'
      - '/Format:hform'
      - '/Format:table'
      - '/Format:mof'
      - '/Format:value'
      - '/Format:rawxml'
      - '/Format:xml'
      - '/Format:csv'
```

Reply:
This has been completed, hoping y'all saw my question below.....